Assets/TextMesh Pro/Documentation/TextMesh Pro User Guide 2016.pdf.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
Do we really need PDF files in the repo ?

Reply:
This is here by default due to textmesh but doing some cleanup might be a good idea

Reply:
Addressed